designer/settings/base.py

```diff
@@ -66,7 +66,6 @@
 
 MIDDLEWARE = (
     'waffle.middleware.WaffleMiddleware',
-    'wagtail.core.middleware.SiteMiddleware',
```

Comment:
I have a concern about this removal. According to documentation at https://docs.wagtail.io/en/stable/releases/2.11.html#sitemiddleware-moved-to-wagtail-contrib-legacy, this is moved to another namespace. That said, I don't see in the code where we use `request.site`. Therefore, I think this might be OK. Will test on stage.